resources/analysisTools/rnaseqworkflow/HIPO2_rnaseq_processing.sh

```diff
@@ -37,9 +37,9 @@ check_executable "$ARRIBA_READTHROUGH_BINARY"
 check_executable "$ARRIBA_DRAW_FUSIONS"
```

Comment:
`check_executable "$ARRIBA_READTHROUGH_BINARY"` can be removed, as this executable no longer exists in Arriba version 1.2.0.